app.py

```diff
@@ -130,3 +131,37 @@ def index():
         logger.critical("Caught unhandled exception.", exc_info=True)
 
     return ""
+
+
+@app.schedule("cron(5 0 * * *)")
```

Comment:
Maybe we don't need to run this often?

Reply:
This will be daily if my cron-foo is good enough :) To me it seems resonable to remind daily until done, but maybe once per month (the first or second) is enough?

Reply:
Yeah you're right. Maybe it makes most sense to run the first week in the month. But running it 365 times a year shouldn't really have a cost impact I guess... So we can keep this as is if no one else have any suggestion

Reply:
This lambda is not running 24/7. It only wakes up in case of a request. Will this code ever run?

Reply:
I assume so, AWS provide a functionality to invoke the lambda based on specified rules. Would expect this to create a cloudwatch event on deploy.
 
See https://aws.github.io/chalice/topics/events#scheduled-events and https://docs.aws.amazon.com/lambda/latest/dg/services-cloudwatchevents.html